usl_models/tests/flood_ml/test_dataset.py

```diff
@@ -86,45 +86,48 @@ def mock_blob_func(blob_name):
     # geospatial will have the labels creeping into a sequence of zeros.
```

Comment:
It'd be good to add a test for `load_dataset`

Reply:
Done.